tokio/src/io/registration.rs

```diff
@@ -167,9 +165,8 @@ impl Registration {
     ///
     /// * `Poll::Ready(Err(err))` means that the registration has encountered an
     ///   error. This error either represents a permanent internal error **or**
-    ///   the fact that [`register`] was not called first.
+    ///   the fact that `register` was not called first.
```

Comment:
i'm talking about this..

Reply:
It seems like registration is now done in the constructor. The equivalent error now would be calling it _after_ `deregister` has been called.

Reply:
Removed all mentions of `register`